File: NDOTestApp/NDOTravelExpenses-Step 6 VB/BusinessClasses/PictureHeader.vb
Imports NDO
Imports System


<NDOPersistent()> _
Public Class PictureHeader
    ' Properties
    Public Property CreationDate() As DateTime
        Get
            Return _creationDate
        End Get
        Set(ByVal value As DateTime)
            _creationDate = value
        End Set
    End Property

    Public Property Name() As String
        Get
            Return _name
        End Get
        Set(ByVal value As String)
            _name = value
        End Set
    End Property


    ' Fields
    Private _creationDate As DateTime
    Private _name As String
End Class

File: Tutorial/NDOTestApp/NDOTravelExpenses-Step 6/BusinessClasses/PictureHeader.cs
using System;
using NDO;

namespace BusinessClasses
{
    [NDOPersistent]
    public partial class PictureHeader
    {
        string name;
        public string Name
        {
            get { return name; }
            set { name = value; }
        }
        DateTime creationDate;
        public DateTime CreationDate
        {
            get { return creationDate; }
            set { creationDate = value; }
        }

        public PictureHeader()
        {
        }
    }
}
